client/src/app/[locale]/(admin)/dashboard/(admin)/[id]/dashboard-id.tsx

```diff
@@ -57,11 +58,11 @@ const Edit = () => {
 
           <ModalAdmin
             isOpen={isOpenAccept}
-            onConfirm={() => {}}
```

Comment:
Mb write todo

Reply:
makes sense, but when connecting to the base it will be done anyway